src/cart/crt_swim.c

```diff
@@ -26,10 +26,22 @@
 #define D_LOGFAC	DD_FAC(swim)
 #define CRT_USE_GURT_FAC
 
+#include <ctype.h>
 #include "crt_internal.h"
 #include "swim/swim_internal.h"
 
 #define CRT_OPC_SWIM_VERSION	0
+#define CRT_SWIM_FAIL_BASE	((CRT_OPC_SWIM_PROTO >> 16) | \
+				 (CRT_OPC_SWIM_VERSION << 4))
+#define CRT_SWIM_FAIL_DROP_RPC	(CRT_SWIM_FAIL_BASE | 0x1)
```

Comment:
I am not sure I like this; it becomes hard to correlate what is in yaml file
instead it is best in my view to pick/reserve some range of swim fault ids, for example from 65000 to 66000

Reply:
Hmm. In this case we should have a single list of those ID's reservation. In which header file this should be defined?
Do we have a recommended logic to allocate ids?

Reply:
There is really no good way of doing this now, so we probably want to start documenting list of fault injection ids/reservations inside of the sample fault injection yaml file that we use.